code/segmentedtickerplant/stplog.q

```diff
@@ -283,6 +283,13 @@ init:{[dbname]
 .dotz.set[`.z.exit;{
   if[not x~0i;.lg.e[`stpexit;"Bad exit!"];:()];
   .lg.o[`stpexit;"Exiting process"];
+
+  if[.stplg.batchmode=`memorybatch;      // Flushing in memory data to disk during unexpected shutdown when batchmode is set to memorybatch
```

Comment:
```suggestion
  // flushing in memory data to disk during unexpected shutdown when batchmode is set to memorybatch
  if[.stplg.batchmode=`memorybatch;
```

this is what I meant by aligning, keep it consistent with other comments in this function